packages/rich-text-editor/test/visual/lumo/rich-text-editor.test.js

```diff
@@ -27,44 +27,74 @@ describe('rich-text-editor', () => {
-      {"attributes":{"bold":true},"insert":"Bold"},
-      {"attributes":{"list":"bullet"},"insert":"\\n"},
-      {"attributes":{"italic":true},"insert":"Italic"},
-      {"attributes":{"list":"bullet"},"insert":"\\n"},
-      {"attributes":{"underline":true},"insert":"Underline"},
-      {"attributes":{"list":"bullet"},"insert":"\\n"},
-      {"attributes":{"strike":true},"insert":"Strike-through"},
-      {"attributes":{"list":"bullet"},"insert":"\\n"},
-      {"insert":"Headings (H1, H2, H3)"},
-      {"attributes":{"list":"bullet"},"insert":"\\n"},
-      {"insert":"Lists (ordered and unordered)"},
-      {"attributes":{"list":"bullet"},"insert":"\\n"},
-      {"insert":"Text align (left, center, right)"},
-      {"attributes":{"list":"bullet"},"insert":"\\n"},
-      {"attributes":{"script":"sub"},"insert":"Sub"},
-      {"insert":"script and "},{"attributes":{"script":"super"},"insert":"super"},
-      {"insert":"script"},{"attributes":{"list":"bullet"},"insert":"\\n"},
-      {"insert":"In addition to text formatting, additional content blocks can be added.\\nBlockquotes"},
-      {"attributes":{"header":3},"insert":"\\n"},
-      {"attributes":{"blockquote":true},"insert":"\\n"},
-      {"insert":"Code blocks"},
-      {"attributes":{"header":3},"insert":"\\n"},{"insert":"<body>"},
-      {"attributes":{"code-block":true},"insert":"\\n"},
-      {"insert":"  <vaadin-rich-text-editor></vaadin-rich-text-editor>"},
-      {"attributes":{"code-block":true},"insert":"\\n"},
-      {"insert":"</body>"},
-      {"attributes":{"code-block":true},"insert":"\\n"},
-      {"insert":"\\n"}
-    ]`;
+    element.value = JSON.stringify(
+      Array.from({ length: 12 }, () => ({
+        insert: 'High quality rich text editor for the web\n',
+      })),
+    );
     await visualDiff(div, 'max-height');
   });
+
+  describe('rich content', () => {
+    it('headings', async () => {
+      element.value = JSON.stringify([
+        { insert: 'Heading 1\n', attributes: { header: 1 } },
+        { insert: 'Heading 2\n', attributes: { header: 2 } },
+        { insert: 'Heading 3\n', attributes: { header: 3 } },
+        { insert: 'Heading 4\n', attributes: { header: 4 } },
+        { insert: 'Heading 5\n', attributes: { header: 5 } },
+      ]);
+      await visualDiff(div, 'rich-content-headings');
+    });
+
+    it('text formatting', () => {
+      element.value = JSON.stringify([
+        { insert: 'Bold\n', attributes: { bold: true } },
+        { insert: 'Italic\n', attributes: { italic: true } },
+        { insert: 'Underline\n', attributes: { underline: true } },
+        { insert: 'Strike-through\n', attributes: { strike: true } },
+        { insert: 'Sub', attributes: { script: 'sub' } },
+        { insert: 'script and ' },
+        { insert: 'super', attributes: { script: 'super' } },
+        { insert: 'script\n' },
+        { insert: '<vaadin-rich-text-editor></vaadin-rich-text-editor>\n', attributes: { code: true } },
```

Comment:
We don't have a dedicated toolbar button for inline code, but IMO it's good to have a test for the format 👍